src/data/python-quiz.ts

```diff
@@ -1084,6 +1084,15 @@ const pythonQuiz = [
     Explanation: "You can use the .join string method to convert a list into a string",
     Link: "https://www.freecodecamp.org/news/python-list-to-string-join-example/",
   },
+  {
+    Question: "exec('a = ( 1 , 2 , 3 , [ 1 , 2 , 3 , 4 , 5 , 6 ] ) \nfor i in range(1,1001): \n\t\t a[3].append(i) \nprint(len(a)) ')",
+    Answer: "4",
+    Distractor1: "Error",
+    Distractor2: "1009",
+    Distractor3: "4",
+    Explanation: "List is mutable in nature so it's size changes",
+    Link: "https://www.freecodecamp.org/news/mutable-vs-immutable-objects-python/",
+  },
```

Comment:
We need to be careful about questions involving code snippets because they don't always render nicely on the page.
<img width="1392" alt="Screen Shot 2022-10-05 at 1 16 40 PM" src="https://user-images.githubusercontent.com/67210629/194154830-9b3dba4a-8144-427f-87e3-73e5fea22fba.png">

Please come up with a new quiz question.